### PR TITLE
Rename TenantID to TenantName

### PR DESCRIPTION
**Pull request by lionello**
## Description

* add `--org` flag

Like `--cluster org@` but that one's hidden for us only.

## Checklist

- [x] I have performed a self-review of my code
- [ ] I have added appropriate tests
- [ ] I have updated the Defang CLI docs and/or README to reflect my changes, if necessary

### CODE REVIEW

File: src/cmd/cli/command/compose.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func makeComposeUpCmd() *cobra.Command {
 				}
 
 				if !resp.Gpu {
-					return ErrNoPermission("usage of GPUs. To resolve see https://s.defang.io/subscription")
+					// return ErrNoPermission("usage of GPUs. To resolve see https://s.defang.io/subscription")
```
Comment (jordanstephens) [inline src/cmd/cli/command/compose.go:93]: Was this intentional? Let's remove the line above and below if we don't want to return an error here.

But if we don't want to return an error here, what's taking its place?

**Review by jordanstephens — APPROVED** [review]

File: src/cmd/cli/command/commands.go [hunk 2]
```diff
@@ -181,8 +190,8 @@ func SetupCommands(ctx context.Context, version string) {
 	RootCmd.PersistentFlags().StringArrayP("file", "f", []string{}, `compose file path`)
 	_ = RootCmd.MarkPersistentFlagFilename("file", "yml", "yaml")
 
-	// Setup tracking client
-	track.Fabric = cli.NewGrpcClient(ctx, cluster)
+	// Client a temporary gRPC client for tracking events before login
+	_ = cli.NewGrpcClient(ctx, cluster)
```
Comment (edwardrf) [inline src/cmd/cli/command/commands.go:194]: nit: assignment not needed.
```suggestion
	cli.NewGrpcClient(ctx, cluster)
```

**Review by edwardrf — APPROVED** [review]

File: src/cmd/cli/command/compose.go (hunk 1 above)
Comment (lionello) [reply src/cmd/cli/command/compose.go]: Good catch. Reverted

File: src/cmd/cli/command/commands.go (hunk 2 above)
Comment (lionello) [reply src/cmd/cli/command/commands.go:194]: Yes but that code without assignment looks wrong

**Pull request merged**